ansible/dump_inventory.py

```diff
@@ -24,13 +24,27 @@ def natural_sort_key(s, _nsre=re.compile('([0-9]+)')):
     return [int(text) if text.isdigit() else text.lower()
             for text in re.split(_nsre, s)]   
 
+def print_host(host):
+        host_ip = host
+        if 'ansible_host' in hostvars[host]:
+            host_ip = hostvars[host]['ansible_host']
+        print("  - %s (%s) [status](http://%s:3013/v2/status) [top](http://%s:3013/v2/blocks/top)" % (host, host_ip, host_ip, host_ip))
+
 for group_name in env_groups:
     print("#### %s\n" % group_name[len(env_group_prefix):])
     hosts = data[group_name]['hosts'].sort(key=natural_sort_key)
     hostvars = data['_meta']['hostvars']
+    seeds = []
+    peers = []
     for host in data[group_name]['hosts']:
-        host_ip = host
-        if 'ansible_host' in hostvars[host]:
-            host_ip = hostvars[host]['ansible_host']
-        print("- %s (%s) [status](http://%s:3013/v2/status) [top](http://%s:3013/v2/blocks/top)" % (host, host_ip, host_ip, host_ip))
+        if 'kind' in hostvars[host]['tags'] and hostvars[host]['tags']['kind'] == 'seed':
+            seeds.append(host)
+        else:
+            peers.append(host)
+    print('seeds:')
```

Comment:
let's add some bold formatting to this sections:
```suggestion
    print('**seeds:**')
```